pkg/storkctl/clusterpair.go

```diff
@@ -285,17 +370,50 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				util.CheckErr(err)
 				return
 			}
+
+			printMsg(fmt.Sprintf("Creating Secret and Backuplocation in source cluster in namespace %v...", cmdFactory.GetNamespace()), ioStreams.Out)
 			storkops.Instance().SetConfig(conf)
+			core.Instance().SetConfig(conf)
+			secret := &v1.Secret{
+				ObjectMeta: meta.ObjectMeta{
+					Name:      clusterPairName,
+					Namespace: cmdFactory.GetNamespace(),
+					Annotations: map[string]string{
+						skipResourceAnnotation: "true",
+					},
+				},
+				Data: credentialData,
+				Type: v1.SecretTypeOpaque,
+			}
+			_, err = core.Instance().CreateSecret(secret)
+			if err != nil {
+				util.CheckErr(err)
+				return
+			}
+
+			backupLocation.Location.SecretConfig = clusterPairName
+			var annotations = make(map[string]string)
+			annotations[skipResourceAnnotation] = "true"
+			_, err = storkops.Instance().CreateBackupLocation(backupLocation)
+			if err != nil {
+				err := fmt.Errorf("unable to create backuplocation in source. Err: %v", err)
+				util.CheckErr(err)
+				return
+			}
+			printMsg(fmt.Sprintf("Backuplocation %v added successfully on source cluster in namespace %v\n", clusterPairName, cmdFactory.GetNamespace()), ioStreams.Out)
+
+			printMsg("Using PX-Service Endpoint of destination cluster to create clusterpair on source cluster...", ioStreams.Out)
```

Comment:
nit: lets change this to : Creating a cluster pair. Direction: Source -> Destination

Reply:
Done